docs/src/man/split_apply_combine.md

```diff
@@ -269,3 +247,40 @@ Last Group (5 rows): g = 501
 │ 4   │ 501   │ 2504  │
 │ 5   │ 501   │ 2505  │
 ```
+
+In order to apply a function to each non-grouping column ofr a `GroupedDataFrame` you can write:
+```jldoctest sac
+julia> gd = groupby(iris, :Species);
+
+julia> combine(gd, valuecols(gd) .=> mean)
```

Comment:
Why not show this using `by`, and put it above, at the same place where `aggregate` was shown?

Reply:
I will add it in https://github.com/JuliaData/DataFrames.jl/pull/2177, as without that PR it is not very easy to do.

Reply:
But I will keep it here as it shows how `valuecols` can be used.